integration-tests/test_client_options.py

```diff
@@ -17,3 +22,53 @@ def test_set_ansible_host_info(insights_client):
     ret = insights_client.run("--ansible-host=foo.example.com", check=False)
     assert "Could not update Ansible hostname" not in ret.stdout
     assert ret.returncode == 0
+
+
+def test_group(insights_client, tmp_path):
+    """Test --group option
+    This test generates an archive in offline mode and verifies if the group
+    specified in --group was created and packed
+    """
+
+    group_name = f"RHIQE.{uuid.uuid4()}.testing-group"
```

Comment:
using an unique UUID is fine, although a bit complicated; maybe simply hardcode something like `testing-group` here?

Reply:
using unique testing group name should be fine because these groups gets created in inventory and incase if test failure it might come handy to look for exact group